dietpi/dietpi-software

```diff
@@ -1032,6 +1032,17 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=9863#p9863'
 		aSOFTWARE_REQUIRES_GIT[$software_id]=1
 		aSOFTWARE_REQUIRES_MYSQL[$software_id]=1
+		#------------------
+		software_id=177
+
+		aSOFTWARE_NAME[$software_id]='Firefox Sync Server'
+		aSOFTWARE_DESC[$software_id]='Sync bookmarks, tabs, history & passwords'
+		aSOFTWARE_TYPE[$software_id]=0
+		aSOFTWARE_CATEGORY_INDEX[$software_id]=4
+		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=42#p42'
+		aSOFTWARE_REQUIRES_BUILDESSENTIAL[$software_id]=1
+		aSOFTWARE_REQUIRES_GIT[$software_id]=1
```

Comment:
Git is actually not required here, the GitHub archive can be downloaded directly, as we do in other cases: https://github.com/mozilla-services/syncserver/archive/master.tar.gz
As long as there is no internal updater that makes use of Git, and one does not plan to develop the code, we can skip it.